src/components/Pagination/Pagination.scss

```diff
@@ -322,7 +324,7 @@ $prefixClass: rc-pagination;
         border: 1px solid transparent;
         white-space: nowrap;
         padding: 0 8px;
-        font-size: 12px;
+        font-size: $font-size;
```

Comment:
@Farhan-emgage I don't think this is necessary. It was already specified on the parent class (line 17)